src/app/_components/Popup.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+
+import Image from 'next/image';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+import { Autoplay, Pagination } from 'swiper/modules';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+const Popup = () => {
+  return (
+    <Swiper
+      slidesPerView={1}
+      loop
+      pagination={{ clickable: true }}
+      autoplay={{
+        delay: 2000,
+        disableOnInteraction: true,
+      }}
+      modules={[Pagination, Autoplay]}
+      className='flex max-w-[350px] flex-col bg-[#212529]'
+    >
+      {[0, 1, 2, 3, 4].map(idx => (
+        <SwiperSlide key={idx}>
+          <Image
+            src={`/SlideD${idx}.webp`}
+            alt={'slide'}
```

Comment:
[제안] 각 이미지별 주요 내용을 alt 에 간략히 적어주시면 접근성 향상에 도움이 될것같아요!

Reply:
좋은 생각이네요

Reply:
```suggestion
            alt= 'slide'
```
```suggestion
            alt={`slide-${idx}`}
```
와 같이 작성해도 좋았을 것 같아요 ! 개인적인 스타일이라고 생각하긴합니다 :)